content/en/network_performance_monitoring/guide/manage_traffic_costs_with_npm.md

```diff
@@ -0,0 +1,64 @@
+
+When we began our Kubernetes migration, migrating stateless services was (expectedly) much faster and easier than migrating our stateful services (e.g. Kafka), so we migrated our stateless services first. The outcome was terabytes of cross-AZ traffic between the stateful services (all in one AZ) and stateless services (spread across the other AZs), but then our cloud bill exploded. We used our own Network Product to identify the root cause - a suboptimal migration strategy - to optimize our network (by sharding our stateful services) and ultimately saw significant reductions in our cloud provider costs.
+
+## Steps for managing traffic costs 
+
+1. To find similar issues in your own environment, you can start by grouping your traffic flows by regions, 
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/region.png" alt="Group flows by region">}}
+    availability zones,
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/availability_zone.png" alt="Group flows by availability">}}
+    and datacenters:
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/datacenter.png" alt="Group flows by datacenter">}}
+    An increase in your traffic bill is almost always tied to an increase in one of these types of traffic. In many cases you may want to group traffic by asymmetric search terms - meaning, you want to see the source of the traffic in terms of one tag, but want to see the destination in terms of another. This kind of asymmetric query can be used to identify costly dependencies between your on-prem datacenters and cloud regions, 
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/aws_account.png" alt="Identify dependencies between datacenters and cloud regions">}}
+    as well as between clouds. One particularly helpful view is to group the source of the traffic by service, and the destination across availability zones. This will help you identify particularly noisy and expensive services that are communicating across multiple availability zones, like what we found during our migration.
+
+2. From here, isolate services that have the most traffic volume across multiple AZs. You can use filters inside the search bars to narrow down your query - so, for example, you can show only services that originate inside of one availability zone, and are sending traffic to any availability zone other than that one.
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/service_availability_zone.png" alt="Highlight any services that communicated across AZs">}}
+    The query above highlights any services that communicate across AZs, from us-east4-a to anywhere else. The chattiest services surfaced by this query may contribute heavily to overall cross-AZ traffic volume, and we can narrow it down even further by searching for one particular service, grouping the traffic by availability zone.
```

Comment:
```suggestion
    The query above highlights any services that communicate across AZs, from us-east4-a to anywhere else. Since the table is already sorted by volume, the first few rows surface the chattiest services contributing to the most cross-AZ traffic. If we want to inspect the cross-infrastructure effects of one of these culprits, we can filter the Source to that particular service and view its traffic to every other availability zone.
```